src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorComponentWithTargetPath.cs

```diff
@@ -10,61 +10,61 @@
-
+        
         /// <summary>The Label given in the Rule.DataSource property.  May not apply to every Property's individual DataSource.</summary>
         internal const string PrimaryDataSourceLabel = "";
-
+        
         /// <summary> (The "TargetPath" property).</summary>
         internal const string TargetPathProperty = "TargetPath";
-
+        
         /// <summary>Backing field for the <see cref='Microsoft.Build.Framework.XamlTypes.Rule'/> property.</summary>
         private Microsoft.VisualStudio.ProjectSystem.Properties.IRule rule;
-
+        
         /// <summary>Backing field for the file name of the rule property.</summary>
         private string file;
-
+        
         /// <summary>Backing field for the ItemType property.</summary>
         private string itemType;
-
+        
         /// <summary>Backing field for the ItemName property.</summary>
         private string itemName;
-
+        
         /// <summary>Configured Project</summary>
         private Microsoft.VisualStudio.ProjectSystem.ConfiguredProject configuredProject;
-
+        
         /// <summary>The dictionary of named catalogs.</summary>
         private System.Collections.Immutable.IImmutableDictionary<string, Microsoft.VisualStudio.ProjectSystem.Properties.IPropertyPagesCatalog> catalogs;
-
+        
         /// <summary>Backing field for the <see cref='Microsoft.VisualStudio.ProjectSystem.Properties.IRule'/> property.</summary>
         private Microsoft.VisualStudio.ProjectSystem.Properties.IRule fallbackRule;
-
+        
         /// <summary>Thread locking object</summary>
         private object locker = new object();
-
+        
         /// <summary>Initializes a new instance of the RazorComponentWithTargetPath class.</summary>
         internal RazorComponentWithTargetPath(Microsoft.VisualStudio.ProjectSystem.Properties.IRule rule)
         {
             this.rule = rule;
         }
-
+        
         /// <summary>Initializes a new instance of the RazorComponentWithTargetPath class.</summary>
-        internal RazorComponentWithTargetPath(Microsoft.VisualStudio.ProjectSystem.ConfiguredProject configuredProject, System.Collections.Immutable.IImmutableDictionary<string, Microsoft.VisualStudio.ProjectSystem.Properties.IPropertyPagesCatalog> catalogs, string context, string file, string itemType, string itemName) :
+        internal RazorComponentWithTargetPath(Microsoft.VisualStudio.ProjectSystem.ConfiguredProject configuredProject, System.Collections.Immutable.IImmutableDictionary<string, Microsoft.VisualStudio.ProjectSystem.Properties.IPropertyPagesCatalog> catalogs, string context, string file, string itemType, string itemName) : 
                 this(GetRule(System.Collections.Immutable.ImmutableDictionary.GetValueOrDefault(catalogs, context), file, itemType, itemName))
         {
-            if (configuredProject is null)
+            if ((configuredProject == null))
```

Comment:
I'm not actually sure... I'll revert. These are generated right? may just be that I didn't run the thing that generates them updated the same way it is in main

Reply:
Oh, this might be my fault. These are generated by CPS, and I had to bump the version for my changes there the other week. These changed for me at the time, but I ended up reverting a bunch of code and trying a different approach for something. I suspect maybe they only get regenerated once, when a new CPS dll is referenced or something. They certainly haven't re-appeared on my machine.

If anyone sees them again, we can probably just commit them.